public/robots.txt

```diff
@@ -0,0 +1,9 @@
+# *
```

Comment:
Should these files be ignored?

Reply:
As I understood it, site.xml and robots.txt are automatically created postbuild, and can therefore be added to .gitignore.. But, as I'm not sure how to set the siteUrl in staging and preview, As I could´t see how sitemap.xml and robots.xml looked in preview, I removed the files from .gitignore to including the files in the public folder. 

From preview, it looks like it works as it should, so I think I can add add the files to .gitignore again.